site/docs/samples/notebook-write.md

```diff
@@ -0,0 +1,405 @@
+while [[ $(kubectl get fybrikapplication my-notebook-write -n fybrik-notebook-sample -o 'jsonpath={.status.assetStates.new-data.conditions[?(@.type == "Deny")].status}') != "True" ]]; do echo "waiting for my-notebook-write asset" && sleep 5; done
+```
+
+It is expected that the asset status is denied due to the policy defined above. Next, a new policy will be applied which will allow the writing to `theshire` object store.
+
+### Cleanup scenario one
+
+Before prceeding to scenario two the OPA policy and fybrikapplications should be deleted:
+
+```bash
+kubectl delete cm sample-policy-write -n fybrik-system
+kubectl delete fybrikapplications.app.fybrik.io my-notebook-write -n fybrik-notebook-sample
+```
+
+Notice that `FybrikStorageAccount` resources are still applied after the cleanup.
+## Scenario two: write new data
+
+To write the new data a new policy should be defined.
+
+### Define data access policies for writing the data
+
+Define an [OpenPolicyAgent](https://www.openpolicyagent.org/) policy to return the list of column names tagged as sensitive whose destination is not neverland when the actionType is write. The columns are passed to the `FybrikModule` together with RedactAction upon deployment of the module by Fybrik.
+Below is the policy (written in [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/#what-is-rego) language):
+
+```rego
+package dataapi.authz
+
+rule[{"action": {"name":"RedactAction","columns": column_names}, "policy": description}] {
+  description := "Redact written columns tagged as sensitive in datasets tagged with finance = true. The data should not be stored in `neverland` storage account"
+  input.action.actionType == "write"
+  input.resource.metadata.tags.finance
+  input.action.destination != "neverland"
+  column_names := [input.resource.metadata.columns[i].name | input.resource.metadata.columns[i].tags.sensitive]
+}
+```
+
+Copy the policy to a file named `sample-policy-write.rego` and then run:
+
+```bash
+kubectl -n fybrik-system create configmap sample-policy-write --from-file=sample-policy-write.rego
+kubectl -n fybrik-system label configmap sample-policy-write openpolicyagent.org/policy=rego
+while [[ $(kubectl get cm sample-policy-write -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
+```
+
+### Deploy a Jupyter notebook
+
+In this sample a Jupyter notebook is used as the user workload and its business logic requires writing the new asset. Deploy a notebook to your cluster: execute the instructions from `Deploy a Jupyter notebook` section in the [notebook sample for the read flow](notebook-read.md) to deploy a Jupyter notebook.
+
+### Create a `FybrikApplication` resource for the notebook
+
+Re-apply `FybrikApplication` resource from scenarios one section.
```

Comment:
Re-apply FybrikApplication as defined in scenario 1.

Reply:
Fixed. Thanks